core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -95,23 +95,22 @@ public static UnderFileSystem create(String path, UnderFileSystemConfiguration u
           // Use the factory to create the actual client for the Under File System
           return new UnderFileSystemWithLogging(factory.create(path, ufsConf));
         } catch (Throwable e) {
-          // This needs to be Throwable rather than Error to catch service loading errors
+          // Catching Throwable rather than Exception to catch service loading errors
           errors.add(e);
-          LOG.warn("Failed to create UnderFileSystemFactory {}", factory, e);
+          LOG.warn("Failed to create UnderFileSystem by factory {}", factory, e.getMessage());
```

Comment:
I think you need to update the message to have another `{}` since `e.getMessage()` is no longer a throwable.